src/save_index.cpp

```diff
@@ -46,6 +46,16 @@ void replace_space2underbar(std::string &name) {
 	std::replace(name.begin(),name.end(),' ','_');
 }
 
+/** Replace the spaces in the filter */
+std::string replace_filter(const std::string* filter)
+{
+    std::string filter_value = *filter;
+    std::string filter_replaced;
+    std::copy(filter_value.begin(), filter_value.end(), filter_replaced.begin());
+    replace_space2underbar(filter_replaced);
```

Comment:
Ok, thank you for the suggestion. Forgive me: I'm not a skilled C/C++ programmer. I primarily use languages like Java, C# etc. where memory management is transparent.